proto/README.md

```diff
@@ -12,15 +12,29 @@ Gradle automatically compiles these proto files, no further action is needed.
 
 To generate:
 
-```
+```sh
 python -m grpc_tools.protoc -I=. --python_out=../rqd/rqd/compiled_proto --grpc_python_out=../rqd/rqd/compiled_proto ./*.proto
 ```
 
+For Windows (Powershell):
```

Comment:
Thanks for adding this to the documentation. 
@sharifsalah, should we add a Windows track for the opencue.io documentation as well once these changes are merged?

Reply:
Sounds good @gregdenton. Yes, we should add a separate Windows track to the getting started guides and a Windows quick start.

I've created https://github.com/AcademySoftwareFoundation/opencue.io/issues/146 to track.

I'm happy to help, but I'm not sure if I'm the best person to write the initial drafts / notes, as I don't have access to a desktop version of Windows. I do have access to Windows server which might help for some of the steps. Also happy to turn basic notes into more polished docs.